kong/utils.go

```diff
@@ -208,12 +208,53 @@ func getConfigSchema(schema gjson.Result) (gjson.Result, error) {
 	return schema, fmt.Errorf("no 'config' field found in schema")
 }
 
+func traverseConfigMap(currentConfigMap map[string]interface{}, path []string) (interface{}, error) {
```

Comment:
Can we add a godoc for this func to explain to the caller how this should be used?